docker-compose-test.yml

```diff
@@ -25,3 +25,23 @@ services:
       context: cirrus/server/
       dockerfile: Dockerfile
     working_dir: /cirrus
+
+  demo-app-server:
+    build:
+      context: ./demo-app/server
+      dockerfile: Dockerfile
+    ports:
+      - '3002:3002'
+    restart: always
+
+  demo-app-frontend:
+    build:
+      context: ./demo-app/frontend
+      dockerfile: Dockerfile
+    ports:
+      - '8080:3000'
+    restart: always
+    links:
+      - demo-app-server
+    environment:
+      - DEMO_APP_SERVER=http://demo-app-server:3002
```

Comment:
Oh no put these in `docker-compose.yml` so they're always available pls 🙏

Reply:
Oh actually I guess we need them both in `docker-compose` and `docker-compose-integration-test` since the two places I think we wanna use it are for local dev and integration testing